tox.ini

```diff
@@ -410,14 +409,6 @@ usedevelop = True
 deps =
     grpcio==1.43.0
     py-eth-sig-utils==0.4.0
-    numpy==1.21.6
```

Comment:
Have you run `tox -e abci-docstrings` locally? It doesn't run in CI so cannot know from CI if this is still working.

Reply:
Yes, still working. Not sure if there is something cached though :thinking: